tests/sentry/hybrid_cloud/test_organization.py

```diff
@@ -286,3 +286,28 @@ def test_get_audit_log_metadata(self):
         self.create_team(organization=org, slug="baz", members=[user])
         rpc_member = serialize_member(member)
         assert member.get_audit_log_data() == rpc_member.get_audit_log_metadata()
+
+
+@django_db_all(transaction=True)
+@region_silo_test(stable=True)
+def test_org_member():
+    org = Factories.create_organization()
+    Factories.create_organization()
```

Comment:
Is this an extraneous org creation, or does this serve some purpose in testing?

Reply:
It's a copy-paste oopsie 😅